packages/wp-now/src/start-server.ts

```diff
@@ -95,5 +96,22 @@ export async function startServer(options: WPNowOptions = {}) {
 
 	app.listen(port, () => {
 		console.log(`Server running at http://127.0.0.1:${port}/`);
+		const porturl = `http://127.0.0.1:${port}`;
```

Comment:
A minor thing, let's use the same variable `porturl` for the console.log
```suggestion
		const porturl = `http://127.0.0.1:${port}`;
		console.log(`Server running at ${porturl}`);
```

Reply:
I made some changes to the code and moved it to a separate function. I think this won't be applied anymore but let me know what you think!